pandas/io/sas/sas_xport.py

```diff
@@ -253,6 +253,9 @@ def __init__(self, filepath_or_buffer, index=None, encoding='ISO-8859-1',
 
         self._read_header()
 
+    def close(self):
+        self.filepath_or_buffer.close()
```

Comment:
It's either a file handle coming from [L244](https://github.com/agraboso/pandas/blob/3fa7d25a00372a1312cd8fad0a82082e997f51a4/pandas/io/sas/sas_xport.py#L244) or a `BytesIO` object coming from [L252](https://github.com/agraboso/pandas/blob/3fa7d25a00372a1312cd8fad0a82082e997f51a4/pandas/io/sas/sas_xport.py#L252). So it is always safe to close it.